db/schema.rb

```diff
@@ -22,8 +22,8 @@
     t.datetime "updated_at", precision: nil
     t.jsonb "events"
     t.datetime "created_at", precision: nil
-    t.check_constraint "created_at IS NOT NULL", name: "application_created_at_null", validate: false
-    t.check_constraint "updated_at IS NOT NULL", name: "application_updated_at_null", validate: false
+    t.check_constraint "created_at IS NOT NULL", name: "application_created_at_null"
```

Comment:
This looks wrong - the `validate: false` got added in AddNotNullConstraintsToTimeStamps.

Reply:
rectified by generated a schema.rb file from a fresh with migration
it removes casting user ids to integers which I'm not sure why was in there in the first place